docs/UserGuide.md

```diff
@@ -209,7 +209,7 @@ and `johndoe@example.com` respectively.
 
 #### Finding a contact : `find`
 
-Finds persons whose names contain any of the given keywords. Also supports search with additional specifiers such as 
+Finds persons whose names contain any of the given keywords. This also supports search with additional specifiers such as 
```

Comment:
The 'This' is not necessary here. Adding it here would only confuse the reader as 'This' could refer to multiple things in this context.

Reply:
I do not think this change is confusing as 'This' clearly refers to the command of finding a contact. 

We need to bear in mind that this UG is also graded by a professor who is not from SOC, and hence may be more particular in terms of grammar and sentence expression.

Reply:
Changed it to `This command`

Reply:
My point is that adding 'This' is a language error. Adding 'This command' is redundant because the entire statement is about the command.